docs/porting.md

```diff
@@ -153,6 +153,21 @@ after importing a module that does not support the GIL.
         endif
         ```
 
+=== "f2py"
+
+    Starting with NumPy 2.1.0 (only available via the nightly wheels or the
+    `main` branch as of right now), Fortran extension modules wrapped using
```

Comment:
The "Fortran extension modules" wording looks very slightly off to me, but I'm also having a hard time rewording it. Perhaps _"extension modules containing f2py-wrapped Fortran code"_ works?